summaries/2020/10097.md

```diff
@@ -2,6 +2,40 @@
 
 https://developer.apple.com/wwdc20/10097
 
-Presenters: _Example Guy, Another Person_
+Presenters: _Steve Breen_
 
-##### TO-DO! You can contribute to this session, please see [CONTRIBUTING.md](CONTRIBUTING.md)
+**Diffable Data Source**
+Recap
+- UICollectionView was first released in iOS 6
+- UICollectionView was built on was the separation of concerns between the data, or the "what," from the layout, the "where" content is being rendered.
```

Comment:
```suggestion
- `UICollectionView` was built on the separation of concerns between the data, or the "what"; from the layout, the "where" content is being rendered.
```

Reply:
I think that if we added ` formatting to all instances of UICollectionView and similar code references, it would look/read better 😄 up to you, though.